app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala

```diff
@@ -79,7 +79,7 @@ class SubscriptionCompleteBusinessService @Inject() (
     } yield (): Unit
 
   private def dataStoreEmailRequest(recipient: RecipientDetailsWithEori)(implicit hc: HeaderCarrier): Future[Unit] =
-    if (recipient.recipientDetails.service == "cds")
+    if (recipient.recipientDetails.service == "HMRC_CUS_ORG")
```

Comment:
This is the only place where "recipientDetails.service" is actually being used.  So this switch in logic is quite straight forward.

Reply:
It should be `HMRC-CUS-ORG`.

I'm wondering if we should name this val in the recipientDetails from service to enrolment key